pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -843,29 +843,7 @@ int uprobe__crypto_tls_Conn_Close(struct pt_regs *ctx) {
     return 0;
 }
 
+// Dummy implementation until we have offset guessing
 static __always_inline void* get_tls_base(struct task_struct* task) {
```

Comment:
I think it is better to remove the goTLS code from prebuilt until we have the implementation